aseprite/PKGBUILD

```diff
@@ -82,21 +82,42 @@ prepare() {
 build() {
 	echo Building Skia...
 	local _skiadir="$PWD/skia/obj"
-	# Flags can typically be found in `src/skia/gn/skia.gni`... but you're kind of on your own
-	env -C skia gn gen "$_skiadir" --args="`printf '%s ' \
-is_debug=false is_official_build=true skia_build_fuzzers=false \
-skia_enable_{pdf,skottie,skrive,sksl}=false \
+	# Flags can be found by running `gn args --list "$_skiadir"` from skia's directory.
+	# (Pipe the output somewhere, there's a LOT of args.)
+	#
+	# The flags are chosen to provide the API required by Aseprite and nothing else (if possible),
+	# so as to reduce the compilation time and final binary size.
+	#
+	# Individual rationales:
+	#   is_official_build: Suggested by the build instructions.
+	#   skia_build_fuzzers: We don't care about them.
+	#   skia_enable_pdf: Not used by Aseprite.
+	#   skia_enable_skottie: Not used by Aseprite.
+	#   skia_enable_sksl: laf seems to want to use it... but no references are made anywhere.
+	#   skia_enable_svg: Not used by Aseprite. It seems it has its own SVG exporter.
+	#   skia_use_lib*_{encode,decode}: Aseprite only loads PNG assets, so only libpng is required.
+	#   skia_use_expat: Only required for the Android font manager and SVGCanvas/SVGDevice.
+	#   skia_use_piex: Not used by Aseprite. Only used for reading RAW files.
+	#   skia_use_xps: Not used outside of Windows.
+	#   skia_use_zlib: Only used for PDF and RAW files.
+	#   skia_use_libgifcodec: Only used for GIFs, which Aseprite doesn't use.
```

Comment:
Aseprite supports `gif`s quite well, unless that's changed in recent versions.

Reply:
They do, but they load the GIFs into the editor; they never pass those to Skia. I checked that disabling `libpng` does produce an error on startup, citing an internal `sheet.png` file. Considering that there are only PNG files there, disabling support for other formats *in Skia* seems reasonable.

Reply:
Gotcha.